src/Library/demos/Screenshot/main.blp

```diff
@@ -0,0 +1,33 @@
+using Gtk 4.0;
+using Adw 1;
+
+Adw.StatusPage {
+  title: "Screenshot";
+  description: _("Take a picture of the screen");
+  margin-top: 48;
+
+  Box {
+    orientation: vertical;
+    halign: center;
+
+    Picture picture {
+      margin-bottom: 12;
+    }
+
+    Button button {
+      icon-name: "screenshooter-symbolic";
+      margin-bottom: 42;
+      width-request: 6;
+      height-request: 48;
```

Comment:
We shouldn't override stanard elements default allocation (size).

Also, I think a label button would be clearer. No need for small icon button.